### ISSUE TITLE
User Creation double-quotes names with `@` symbol

### ISSUE THREAD

**Issue by dexhorthy** [Titan-Systems/titan#68]
```yaml
# titan.yml

users:
  - name: matt@metalytics.dev
```

This creates the user with extra quotes, 

| Name | Created | Login_name |
|-------| --------| ------------|

![image](https://github.com/Titan-Systems/titan/assets/3730605/09158681-f0e9-43c2-820a-e0439b8f9309)

and in order to log in as this user, need to do something like

```
SNOWSQL_USER='"matt@metalytics.dev"'
```

**Comment by SwapnilSonker** [comment]
do we need to remove the double quotes from the name? , in order to log in as the user ? 
@dexhorthy

**Comment by teej** [comment]
Oof yeah this looks like a bug. This should've been ported to `ResourceName` but it wasn't.

https://github.com/Titan-Systems/titan/blob/main/titan/resources/user.py#L37-L42

You can get around it by specifying `login_name`. I'll get it fixed in the next release

**Comment by dexhorthy** [comment]
confirmed this is fixed in 0.8.1 - thanks!
